assets/lang/es-es.cfg

```diff
@@ -4,16 +4,16 @@ Localization
 	{
 		astrogator_mainTitle = Astrogador
 		astrogator_versionFormat = v<<1>>.<<2>>.<<3>>
-		astrogator_mainTooltip = Sumario de transferencias a los cuerpos accesibles
+		astrogator_mainTooltip = Sumario de ventanas de transferencias a los cuerpos accesibles
 		astrogator_normalSubtitle = Transferencias desde <<1>>
 		astrogator_inboundHyperbolicWarning = <<1>> está en una trayectoria de escape.\nCaptura para ver más transferencias.
 		astrogator_outboundHyperbolicError = <<1>> está en una trayectoria de escape.\nCaptura para ver transferencias.
 		astrogator_landedError = <<1>> está aterrizado. Ponte en órbita para ver las transferencias.
 		astrogator_highInclinationError = La inclinación es <<1>>°, la precisión es demasiado baja pasados los <<2>>°
 		astrogator_noTransfersError = No hay transferencias disponibles
 		astrogator_launchSubtitle = Transferencias desde <<1>>\n(Lanzamiento ~<<2>>)
-		astrogator_transferColumnHeader = Cuerpo
-		astrogator_timeColumnHeader = Tiempo hasta ventana de transferencia
+		astrogator_transferColumnHeader = Transferir
```

Comment:
"Transferencia Hohmann". In fact, "Transferencia" was my first choice there, but I find it too long 😋. Definitely it's the correct translation in that case.

Salud.